bot/seasons/evergreen/__init__.py

```diff
@@ -5,3 +5,9 @@ class Evergreen(SeasonBase):
     """Evergreen Seasonal event attributes."""
 
     bot_icon = "/logos/logo_seasonal/evergreen/logo_evergreen.png"
+    icon = [
```

Comment:
Is there a reason to make this a separate variable? Also, this should be a tuple rather than a list.

Reply:
All seasons have an `icon` attribute and I thought it'd be better to change the evergreen icon and leave the default icon in `season.py` to any new season that may not have an icon yet

Reply:
So `bot_icon` is the default for seasons? That's fair.